flexget/components/sites/sites/newznab.py

```diff
@@ -102,14 +106,12 @@ def do_search_tvsearch(self, arg_entry, task, config=None):
         ):
             return []
         if arg_entry.get('tvrage_id'):
-            lookup = '&rid=%s' % arg_entry.get('tvrage_id')
+            params = {'rid': arg_entry.get('tvrage_id')}
         else:
-            lookup = '&q=%s' % quote(arg_entry['series_name'])
-        url = (
-            config['url']
-            + lookup
-            + '&season=%s&ep=%s' % (arg_entry['series_season'], arg_entry['series_episode'])
-        )
+            params = {'q': arg_entry['series_name']}
+        params['season'] = arg_entry['series_season']
+        params['ep'] = arg_entry['series_episode']
+        url = f"{config['url']}{urlencode(params)}"
```

Comment:
it's be best if you passed params as its own dict and then you wouldn't have to take care of the econding or concating the url. Let reqeuest handle that by passing it to the `params` keyword